Lib/test/test_typing.py

```diff
@@ -2460,6 +2468,17 @@ def test_subclass_special_form(self):
                     class Foo(obj):
                         pass
 
+    def test_or_and_ror(self):
+        self.assertEqual(
+            SimpleMapping[str, int] | bool,
+            Union[SimpleMapping[str, int], bool],
+        )
+        self.assertEqual(
+            bool | SimpleMapping[str, int],
+            Union[bool, SimpleMapping[str, int]],
+        )
+
+
```

Comment:
I don't think we need these tests. Isn't `_GenericAlias` already tested at https://github.com/python/cpython/blob/main/Lib/test/test_types.py#L642 ? Subscripting almost any type in typing returns a `_GenericAlias`.